meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: tomopy
-  version: '0.1.8'
+  version: '0.1.9'
 
 source:
   git_url: https://github.com/dgursoy/tomopy.git
```

Comment:
I haven't thought about using old versions for now. How about using ``git_tag`` instead of ``git_rev``? Seems more cleaner.